### ISSUE TITLE
Problems typing in the answer

### ISSUE THREAD

**Issue by ghost** [Quip13/No-Distractions-Full-Screen#24]
Hello. First of all, great addon! 

Until now it worked great. I recently switched to new device and in full screen mode when I have "type in answer"cards I have to click on typing box every time a new card appears. It does not happen when in windowed mode so it is not about anki itself.

Thank you in advance :)

**Comment by ccz-2** [comment]
Thanks for the report! I'm unable to replicate this issue so could you fill me in:
1) Could you verify that you're on the latest version of NDFS (v4.1.8), version number should be visible in the addon config, accessible from the NDFS menu under "Advanced Settings"
2) What platform are you on? (Mac, windows, linux)
3) When you say it works in "windowed mode", do you mean NDFS windowed mode, or vanilla anki windowed mode
4) Did you not have this issue on your old machine?

**Comment by ghost** [comment]
1. I am on the latest version of NDFS.
2. I am using Windows.
3. I meant vanilla anki windowed mode. On NDFS's windowed mode it is also happening.
4. I did not have this issue on my old machine.

I tried switching the addon on and off. I also tried reinstalling. Nothing helped. I hope I am not a big trouble for you.

**Comment by ghost** [comment]
It might also be worth mentioning that I also use Fill the blank addon for cloze deletion and in "cloze" card type I do not have to click on the box.

**Comment by ghost** [comment]
One more thing worth mentioning: I use flash version of Anki on USB.

I followed this guide here: https://gaijinsunny.wordpress.com/2013/12/21/anki-usb-syncing/

**Comment by ccz-2** [comment]
Thanks for the info! Could you try disabling all other addons except for NDFS and see if the issue still persists?
I'm curious if this is a compatibility issue with another addon

**Comment by ghost** [comment]
I disabled all the other add ons and the problem still persists. Do you want a gif or you get the gist what I mean? 

When I go into full screen mode there is a blink of white color at the bottom of the screen.

**Comment by ghost** [comment]
I looked for the answer on internet and these are the best things I have found.

https://www.geeksforgeeks.org/python-focus_set-and-focus_get-method/

https://doc.qt.io/qtforpython/overviews/eventsandfilters.html

**Comment by ccz-2** [comment]
Yes! A gif would be helpful. This was an issue in the past that I thought I
fixed but this new issue seems a little different

**Comment by ghost** [comment]
I could not create a gif that is small enough. So it is about typing field. In Anki windowed mode it is like this aotomatically (I do not have to click on the box whenever new card appears):

![image](https://user-images.githubusercontent.com/70767944/92334079-f680a300-f08a-11ea-9067-716fb963eb41.png)

In full screen it is like this (I have to click on the box every time a new card appears):

![image](https://user-images.githubusercontent.com/70767944/92334098-20d26080-f08b-11ea-9e3b-684385e13b4b.png)

**Comment by ghost** [comment]
FYI: I tried using it not as a slash version mentioned above, but also as a desktop version installed on my desktop and the problem does not cease to exist :(

**Comment by kcharecki** [comment]
I can confirm that in my case it behaves as @mateuszschmidt mentioned. The input element is not targetted on card load. I need to either manually click it or press TAB 5 times.

**Comment by ghost** [comment]
I have the same problem(
